readme/template-readme.md

```diff
@@ -0,0 +1,101 @@
+# README Template
+
+*{Before using this template, read the accompanying README template guide}.*
+
+## {Project} logo and badges
+{This section is optional. Share your logo and add relevant badges to your README.}
```

Comment:
I would substitute the pronoun "your" with a more concrete "project's":

Add the project's logo and relevant badges to your README.